src/compiler/utilitiesPublic.ts

```diff
@@ -956,19 +957,22 @@ export function getModifiers(node: HasModifiers): readonly Modifier[] | undefine
     }
 }
 
-function getJSDocParameterTagsWorker(param: ParameterDeclaration, noCache?: boolean): readonly JSDocParameterTag[] {
-    if (param.name) {
-        if (isIdentifier(param.name)) {
-            const name = param.name.escapedText;
-            return getJSDocTagsWorker(param.parent, noCache).filter((tag): tag is JSDocParameterTag => isJSDocParameterTag(tag) && isIdentifier(tag.name) && tag.name.escapedText === name);
+function getJSDocParameterTagsWorker(node: ParameterDeclaration | BindingElement, noCache?: boolean): readonly JSDocParameterTag[] {
+    const declaration = getRootDeclaration(node);
+    if (isParameter(declaration)) {
+        const name = node.name;

```

Comment:
I think `name` refers to [renamed identifier](https://ts-ast-viewer.com/#code/GYVwdgxgLglg9mABMMAKA3oghgLkWAUwHcBBRAXwEpF1yg)

Reply:
Yes, in the example above, `name` is `newA` and `propertyName` is `a`. So I think the @param annotation should use `a` instead of `newA`, right? So it should be using the `propertyName` identifier in that case, right?

Reply:
I thought we needed to use a new name :(. Fixed. Thanks for the help.